src/asynch.rs

```diff
@@ -63,7 +59,7 @@ impl<SPI> SmartLedsWriteAsync for Apa102Async<SPI>
 where
     SPI: SpiBus,
 {
-    type Color = RGB8;
```

Comment:
Could you also retain the old implementation so a user can keep on directly using RGB8. Would also make this PR a non-breaking change

Reply:
`impl From<RGB8> for Apa102Pixel` is implemented and the trait bound for `SmartLedsWrite::write` was already specified as `Into<Self::Color>`, so this is not a breaking change for downstreams passing iterators of RGB8 into `SmartLedsWrite::write`.

Reply:
This is unfortunately not equivalent, as while `RGB8` continues to work, other types which implement `Into<RGB8> ` will not as `Into` and `From` are not transitive.

Reply:
Ah, good point. If users aren't using RGB8 directly, but something that also `impl From<RGB8>`, they'll need to add a `.into()` (or `.map(Into::into)`) on their side before passing to `write`. But I suspect almost all users have been providing data as `[RGB8]`.

Reply:
I have added a doctest to lib.rs (run it with `cargo test --doc`) demonstrating that the old way of passing `[RGB8]` continues to work with no change required for the downstream library user.